### DISCUSSION TITLE
Welcome to C-Plus-Plus Discussions!

### DISCUSSION THREAD

**Discussion by Panquesito7** [TheAlgorithms/C-Plus-Plus#1457] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Harshu467** [comment]
Really useful content for people like me in college for learning such things. Loved the content

**Comment by VACALUVISA** [comment]
Pero si nadie comenta nada, desde hace 4 meses, al menos pongan un proyecto para hacer entre varios o alguna vaina.

**Reply by Panquesito7** [reply]
Esto no se usa muy a menudo. Hay mucha más actividad en el servidor de Discord: https://the-algorithms.com/discord
